### DISCUSSION TITLE
OSC Mobile controller

### DISCUSSION THREAD

**Discussion by pprchal** [hydrogen-music/hydrogen#1182] [Q&A]
Beacuse I need to control hydrogen when I'm busy with guitar.
So I created simple project:
https://github.com/pprchal/hydropad

question:
can be this piece usefull to somebody, or there is already better solution - I'm too lazy ;)

Thanks!

**Comment by trebmuh** [comment]
Added to [the wiki](https://github.com/hydrogen-music/hydrogen/wiki/external-software-&-utilities), thanks!

**Comment by mauser** [comment]
> question:
> can be this piece usefull to somebody, or there is already better solution - I'm too lazy ;)

Hi! 

there are also other solutions (not necessarily better). You could use an app like Touch OSC or "Control", which offer to build your own interfaces without the need for programming skills. An example project for TouchOSC can be found here: https://github.com/mauser/HydrogenOSCPresets. I always wanted to write an article for the Hydrogen website about this, but never really found the time to write it up cleanly.. 
 
There is also a preset for Open Stage Control, which is quite similiar to your approach (it uses a webserver instead of a dedicated UI).

**Reply by pprchal** [reply]
Firstly a look for some OSC-mobile app - but without success (poor quality and/or paid sw)

My solution utilizes mobile-web browser, so you are compatible with most devices.
And - I don't want to build user interface , if you're play with band, play/pause tempo up/down is sufficient enough.
But inform me, I'm lazy ;)

P.S:Open Stage Control is reason to this program - too complicated
